src/Mvc/Mvc.Core/src/Routing/ActionEndpointFactory.cs

```diff
@@ -455,7 +455,7 @@ private static RequestDelegate CreateRequestDelegate()
                 }
 
                 var invoker = invokerFactory.CreateInvoker(actionContext);
-                return invoker.InvokeAsync();
+                return invoker!.InvokeAsync();
```

Comment:
This seems fine for now. My guess is this should throw if the invoker cannot be produced. This could happen if a user removed services required to construct a invoker for an AD. FYI @davidfowl